content/docs/get-started/index.md

```diff
@@ -216,84 +194,74 @@ y_pred = model.predict_proba(x)
 print(y_pred)
 ```
 
-Now, let's run the script:
+You can run it to print the prediction probabilities:
 
 ```cli
 $ python predict.py
 [[0.47 0.24 0.29]]
 ```
 
-We see that the prediction probabilities were successfully printed to stdout.
-
 </tab>
 
-<tab title="Command Line">
+<tab title="Command line">
 
 ### Batch scoring
 
-The MLEM CLI allows us to natively use any saved model directly for prediction
-or batch scoring with any local dataset. This is very handy if we want to get
-some quick feedback about a model we just created.
-
-First, create an example dataset file to apply, we'll go with a `csv` format:
+`mlem` lets you use any saved model natively for prediction or batch scoring
+with any dataset. This is very handy if you want to get quick feedback about a
+model, for example. Let's create a sample dataset and apply it to the `rf` model
+we saved earlier:
 
-```cli
-$ echo "sepal length (cm),sepal width (cm),petal length (cm),petal width (cm)
-0,1,2,3" > new_data.csv
+```csv
+sepal length (cm),sepal width (cm),petal length (cm),petal width (cm)
+0,1,2,3
 ```
 
-Next, simply run `mlem apply` to apply this dataset against our model's
-`predict_proba` method:
+Save the contents above in a file called `new_data.csv` and `mlem apply` itit to
+our model's `predict_proba` method as shown below to print the prediction
+probabilities:
 
 ```cli
 $ mlem apply models/rf new_data.csv \
     --method predict_proba \
-    --import \
-    --import-type "pandas[csv]"
+    --import --import-type "pandas[csv]"
 ⏳️ Importing object from new_data.csv
 ⏳️ Loading model from models/rf.mlem
 🍏 Applying `predict_proba` method...
 [[0.47 0.24 0.29]]
 ```
 
-And we get our expected prediction probabilities as output.
-
 <details>
 
-#### Learn more about the CLI options used
+#### Click to learn more about the command options used.
```

Comment:
`.` at the end looks strange to me - you suggest to click on the button, but the sentence ends with the dot. If this is our standard, you can ignore my opinion though

Reply:
Full sentences should end in `.` normally 🙂 

<img width="552" alt="image" src="https://user-images.githubusercontent.com/1477535/209634682-47516e41-dfc7-4b95-812e-4fa01b2e14a5.png">

> https://dvc.org/doc/start

<img width="1012" alt="image" src="https://user-images.githubusercontent.com/1477535/209634754-75779629-8bef-4694-835a-7e125478c048.png">

> https://dvc.org/doc/start/data-management/data-versioning

Ignoring 😬